### DISCUSSION TITLE
Auto submit form when set data using @this.set()

### DISCUSSION THREAD

**Discussion by parthkapatel** [livewire/livewire#7336] [7. Bugs (archived)] [answered]
### Livewire version

3.1

### Which PHP version are you using?

PHP 8.2

### Additional information

Hello @joshhanley ,
I tried your suggested solution for update data from backed using jQuery like https://github.com/livewire/livewire/issues/2206#issuecomment-744066235 but after that I faced one another problem that when this line of code is run then form submitted automatically without clicking on the submit button and page data is load accordingly.

I want to update the multiple dropdown value from the backend and when I'll click on submit button then filter the data according

```
$(document).initMutlipleSelect('multipleSelect',"#level, #admin_status, #access",{
    selectAll: false,
    filter: false,
    placeholder: false
}, function(id, value) {
    console.log("Element ID: ", id);
    console.log("Selected values: ", value);
    @this.set('filters.'+id, value);
});
```

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by JeffGepiga** [comment, marked as answer]
try to add false as third parameter

`@this.set('key',value, false)`

**Reply by parthkapatel** [reply]
Thanks. It's solved my problem
